src/rosdiscover/recover/call.py

```diff
@@ -77,13 +78,13 @@ def is_unknown(self) -> bool:
 @attr.s(frozen=True, auto_attribs=True, slots=True)
 class Publish(SymbolicRosApiCall):
     publisher: str
-    path_condition: t.Dict[str, t.Any]
+    path_condition: SymbolicExpr
```

Comment:
minor suggestion: `condition` may be a more appropriate term than `path condition` in this context

Reply:
fixed